scripts/build.js

```diff
@@ -56,7 +56,7 @@ executeTasks(firstTasks).then(() => {
     process.exitCode = 1;
   }
   logEndBuild(packageName, !hasFailures, buildStartTime);
-});
+}).then(() => executeTask());
```

Comment:
This would run all the time for every build including developer builds. Could you please instead make it trigger in a more discriminate fashion? e.g. off presence of environment variables, CLI commands, arguments.

Reply:
This is a good idea. This should probably look something like:

```JavaScript
.then(() => {
  if (process.env['CI_BUILD']) {
    generateSizeData();
  }
});
```

And add this to the appveyor config file:

```YAML
environment:
  CI_BUILD: TRUE
```

Reply:
See this doc about env variables in AppVeyor: https://www.appveyor.com/docs/build-configuration/#environment-variables